setup/_update_dep_errors.rst.inc

```diff
@@ -6,7 +6,7 @@ other Symfony dependencies too. In that case, try the following command:
 
 .. code-block:: terminal
 
-    $ composer update symfony/symfony --with-dependencies
+    $ composer update "symfony/*" --with-dependencies
```

Comment:
In Composer 1.6, apparently there is a `--with-all-dependencies`. This looks like what we want.

But then, because this new flag works the way we want it to, the user should never have version constraint problems anymore. So, we can add this flag to the other commands, then probably remove this section.